### PR TITLE
add isSubstrateV2

### PR DESCRIPTION
**Pull request by Tbaut**
- as per https://github.com/polkadot-js/apps/pull/1397#discussion_r303215263 this gives an easy way to check on the UI side if we are connected to a Substrate v2
- test is passing for Alex (and failing with a localhost v2 as expected)

### CODE REVIEW

File: packages/api/src/Base.ts [hunk 1]
```diff
@@ -474,6 +476,10 @@ export default abstract class ApiBase<ApiType> {
           this.emit('ready', this);
         }
 
+        if (!Object.keys(this.consts).length) {
+          this._isSubstrateV2 = false;
```
Comment (jacogr) [inline packages/api/src/Base.ts]: I believe this is the other way around?

Comment (Tbaut) [reply packages/api/src/Base.ts]: It's maybe a little hard to read, but if the length is 0 it's a v1, the `!` will make it go into the `if` to set it to `false`.

**Comment by Tbaut** [comment]
I re-read your comment too late in https://github.com/polkadot-js/apps/pull/1397#discussion_r303215263, you wrote `ui-api` from the start. I'll put it there

File: packages/api/src/Base.ts (hunk 1 above)
Comment (jacogr) [reply packages/api/src/Base.ts]: You are right :)

**Pull request closed without merging**